pkgs/top-level/python-packages.nix

```
@@ -11799,6 +11799,11 @@ self: super: with self; {
 
   tabview = callPackage ../development/python-modules/tabview { };
 
+  taco = toPythonModule (callPackage ../development/libraries/taco {
+    inherit python3;
```

```suggestion
  taco = toPythonModule (pkgs.taco.override {
    inherit (self) python;
```
Does this work? That construction is widely used in `python-packages.nix`.